ddtrace/contrib/pytest/_plugin_v1.py

```diff
@@ -582,6 +594,10 @@ def pytest_runtest_protocol(item, nextitem):
         yield
         return
 
+    if PYTEST_CPP_ENABLED and isinstance(item, pytest_cpp.plugin.CppItem):
```

Comment:
I don't love the idea of modifying our `pytest` integration just so we can use a plugin in our CI.

Why do `pytest_cpp` tests need to be ignored?

Reply:
`pytest_cpp.plugin.CppItem` objects don't have member attributes as expected in our `pytest` integration, and I had to add bunch of lines checking for `None` in the code to avoid failures like 
https://gitlab.ddbuild.io/DataDog/apm-reliability/dd-trace-py/-/jobs/699959248

The changes looked like [`e2f518a` (#11108)](https://github.com/DataDog/dd-trace-py/pull/11108/commits/e2f518aaed11295955ad694dac9d6ecbf54999cd)

I wanted to minimize the diff for this PR and reverted above changes and added this line.

Reply:
At first glance, I think most of those will be fixed with the next version of the `pytest` integration (which you can enable with `_DD_CIVISIBILITY_USE_PYTEST_V2=1` in your environment).

I'm actually looking to make the next version of the plugin be the default in `dd-trace-py` for dogfooding reasons, so it'd be a good test.

Reply:
Good to know and will try that, thanks!

Reply:
It works, thank you.